### PR TITLE
Kaung: JavavsCPP - Fixed table title and added new section

### LINKED ISSUE #20: JavavsCPP: Add missing '+' to table comparing two images
In the table with the two images comparing C++ and Java, the title for the C++ image is missing a '+'. I think this should be fixed for aesthetic purposes.  
image

### PR DESCRIPTION
**Pull request by Kaung-Sone**
Fixes #20 and adds new section under "In Java, there is no pointer notation"

### CODE REVIEW

File: docs/labs/JavavsCPP.md [hunk 1]
```diff
@@ -157,6 +157,29 @@ you created a variable that holds a pointer only,
...
+The side effect of having no pointer locations is that you cannot pass parameters into a function by
+reference in Java.
+C++ supports passing parameters both by value and reference, so you can have multiple outputs in a function.
+
+```cpp
+void myFunction(
+  int a, // in
+  int* b, // in
+  int* c, // out
+  int* d, // out
```
Comment (onzfonz) [inline docs/labs/JavavsCPP.md:171]: what are the comments in here?
Did you take this from somewhere in particular? 
If so, we need to cite it

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please also take out the word issue to link to it

File: docs/labs/JavavsCPP.md (hunk 1 above)
Comment (Kaung-Sone) [reply docs/labs/JavavsCPP.md:171]: It is to signify which variables are used as inputs and which variables are used as outputs. I came up with the example myself and did not take it from anywhere. Do you want me to update those comments for extra clarification?

File: docs/labs/JavavsCPP.md [hunk 2]
```diff
@@ -157,6 +157,29 @@ you created a variable that holds a pointer only,
...
 
+### Functions, passing by reference and passing by value
+
+The side effect of having no pointer locations is that you cannot pass parameters into a function by
+reference in Java.
+C++ supports passing parameters both by value and reference, so you can have multiple outputs in a function.
+
+```cpp
+void myFunction(
+  int a, // in
```
Comment (onzfonz) [inline docs/labs/JavavsCPP.md:168]:
```suggestion
  int a,
```

**Review by onzfonz — CHANGES_REQUESTED** [review]

File: docs/labs/JavavsCPP.md [hunk 3]
```diff
@@ -157,6 +157,34 @@ you created a variable that holds a pointer only,
...
+  int a;
+  int* b
+  
+  // Output variables
+  int* c, d;
+  
+  myFunction(a, &b, &c, &d);
+}
+```
+
```
Comment (onzfonz) [inline docs/labs/JavavsCPP.md:185]:
```suggestion
```

**Review by onzfonz — CHANGES_REQUESTED** [review]

**Review by onzfonz — APPROVED** [review]

**Pull request merged**